integrationlink/manual_deploy.go

```diff
@@ -19,11 +20,24 @@ func ManualDeploy(ctx context.Context, app, branch string) error {
 		return errgo.Notef(err, "fail to get Scalingo client")
 	}
 
-	err = c.SCMRepoLinkManualDeploy(ctx, app, branch)
+	deploy, err := c.SCMRepoLinkManualDeploy(ctx, app, branch)
 	if err != nil {
 		return errgo.Notef(err, "fail to trigger manual deploy")
 	}
 
 	io.Statusf("Manual deployment triggered for app '%s' on branch '%s'.\n", app, branch)
+
+	if !follow {
+		return nil
+	}
+
+	err = deployments.Stream(ctx, &deployments.StreamOpts{
+		AppName:      app,
+		DeploymentID: deploy.ID,
+	})
+	if err != nil {
+		return errgo.Mask(err, errgo.Any)
```

Comment:
convention: We no longer use `errgo.Mask`
```suggestion
		return errgo.Notef(err, "stream deployment logs")
```